src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/MaintenanceController.php

```diff
@@ -66,6 +69,7 @@ public function indexAction(Request $request, FormInterface $form = null)
             'requireFilterStatus' => false,
             'generalForm' => $form->createView(),
             'currentIp' => $request->getClientIp(),
+            'isMultishopConfigEnabled' => $isMultishopActive && !$isAllContext,
         ]);
```

Comment:
You are adding isMultishopConfigEnabled, but I can't find where it is being used.